_playbook/pdf.md

```diff
@@ -17,7 +17,9 @@ roles:
   - UX designer
 ---
 
-Most PDFs simply do not meet the Section 508 requirements. It is just simply so easy to produce PDFs, but so difficult to produce accessible ones. There are very few tools that allow you to tell the difference between. One free tool and easy tool is the [Tingtun Checker PDF Checker](http://checkers.eiii.eu/en/pdfcheck/), but does not show all errors that could be there. There are other tools that can provide more detailed results but require installation.
+Most PDFs simply do not meet the Section 508 requirements. It is just simply so easy to produce PDFs, but so difficult to produce accessible ones. There are very few tools that allow you to tell the difference between. 
+
+The Luxembourg government has several open source accessibility projects inclucing their [simplA11yPDFCrawler](https://github.com/accessibility-luxembourg/simplA11yPDFCrawler).This small government has created a great tool which collects and scans PDFs for known accessibility errors. [VeraPDF](https://verapdf.org/home/) or [VeraPDF on GitHub](https://github.com/verapdf) is another option open source option that can provide more detailed results has a more complicated installation. 
```

Comment:
I think the last sentence here is missing a word? And it has the word "option" twice.

`is another *option* open source *option* that can provide more detailed results has a more complicated installation.`

maybe should be "more detailed results *but* has a more complicated installation"?